src/fides/api/custom_types.py

```diff
@@ -28,6 +31,53 @@ def validate(cls, value: str) -> str:
         return value
 
 
+class HtmlStr(str):
+    """
+    This class is designed to be used in place of the `str` type
+    any place where user inputted HTML text is expected.
+
+    The validation applied here enforces that only a subset of "safe" HTML is
+    supported to prevent XSS or similar attacks.
+    """
+
+    # Allow only basic markup tags, for extra safety
+    ALLOWED_HTML_TAGS = {
+        "a",
+        "blockquote",
+        "br",
+        "div",
+        "em",
+        "h1",
+        "h2",
+        "h3",
+        "h4",
+        "h5",
+        "h6",
+        "i",
+        "li",
+        "ol",
+        "p",
+        "pre",
+        "q",
+        "rt",
+        "ruby",
+        "s",
+        "span",
+        "strong",
+        "u",
+    }
+
+    @classmethod
+    def __get_validators__(cls) -> Generator:  # pragma: no cover
+        yield cls.validate
+
+    @classmethod
+    def validate(cls, value: str) -> str:
+        """Assert text doesn't include an complex/malicious HTML."""
+        cleaned_str = clean(value, tags=HtmlStr.ALLOWED_HTML_TAGS)
```

Comment:
two nits:
- still working on my python conventions but pretty sure the convention here is to use the `cls` self-reference arg in the classmethod 👍 (see suggestion)
- you didn't wanna keep the null/false check on `value` that you had in the previous iteration? i know that technically the edge case may be handled by `clean()` but feels like a decent defensive measure and to my eyes, it helps readability/reduces confusion, especially since this is a 3rd party lib. but i can see arguments for why this approach may be _cleaner_ (pun intended 😉)


```suggestion
        cleaned_str = clean(value, tags=cls.ALLOWED_HTML_TAGS)
```

Reply:
Thanks! Both good suggestions, will update now...